### PR TITLE
#182631413-PAGINATION SUPPORT FOR ARTICLES

### PR DESCRIPTION
**Pull request by ChrisOnsando**
#### What does this Pull Request do?
- [ ] Creates pagination for articles 
#### Description of Task to be completed?
- [ ] Create pagination for articles
#### How should this be manually tested?
- [ ] The articles will be viewed per the pagination specification
#### Any background context you want to provide?
- [ ] In most cases a user would want quick access to the most recent articles  while the older articles are only fetched when the user scrolls up/down or selects the next page, pagination eases the scrolling for the user
#### What are the relevant Pivotal Tracker Story?
- [ ] https://www.pivotaltracker.com/story/show/182631413
#### Screenshots (if appropriate)
Screenshot from 2022-08-03 12-31-14
#### Questions:
- [ ] None

### CODE REVIEW

File: speaksfer/settings/base.py [hunk 1]
```diff
@@ -156,6 +156,8 @@
         "rest_framework.renderers.MultiPartRenderer",
         "rest_framework.renderers.JSONRenderer",
     ),
+    "DEFAULT_PAGINATION_CLASS": "rest_framework.pagination.PageNumberPagination",
+    "PAGE_SIZE": 2,
```
Comment (mainamuriuki274) [inline speaksfer/settings/base.py:160]: increase the page size

**Review by mainamuriuki274 — CHANGES_REQUESTED** [review]

File: speaksfer/settings/base.py (hunk 1 above)
Comment (ChrisOnsando) [reply speaksfer/settings/base.py:160]: To what size?

Comment (ChrisOnsando) [reply speaksfer/settings/base.py]: I have increased the page size

**Comment by dalienst** [comment]
I have reviewed the code. 
It is good.

**Review by mainamuriuki274 — APPROVED** [review]

**Pull request merged**